assets/app/styles/_components.less

```diff
@@ -204,20 +204,23 @@
 }
 
 .pod-block {
-  .scaling-controls {
```

Comment:
This is arguably a problem with a lot of our styles. It makes it difficult to restructure the markup if you want. I remember reading on the Mozilla site that it's bad for CSS performance as well.

I want to eventually have this chart on the deployment page, so I think it makes sense to move out.